data/data/gcp/variables-google.tf

```diff
@@ -1,9 +1,36 @@
-variable "google_project" {
```

Comment:
https://github.com/openshift/installer/pull/1938/commits/388c54599c9c4995a521225a7064f2698475ce04

for making sure the prefix is gcp not google